src/common/timer_manager.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <asio/io_service.hpp>
+#include <iostream>
+#include <memory>
+#include <thread>
+#include <vector>
+
+#include "common/randomer.h"
+#include "common/timer.h"
+
+namespace raftcpp {
+namespace common {
+
+/**
+ * The manager class to manage all of the timers for node.
+ */
+class TimerManager final {
+public:
+    explicit TimerManager();
+
+    ~TimerManager();
+
+    void Run();
+
+    int RegisterTimer(const std::function<void(void)> &handler);
```

Comment:
Could you add some document comments on the methods?

Reply:
OK